xclim/run_length.py

```diff
@@ -460,7 +460,7 @@ def first_run_ufunc(
         kwargs={"window": window},
     )
 
-    if index is not None and ~(np.isnan(ind)):
+    if index is not None and np.all(~(np.isnan(ind))):
```

Comment:
Would `ind.notnull().all()` be clearer ?

Reply:
Yes, definitely!